src/useDirectUpload.ts

```diff
@@ -38,12 +38,14 @@ const useDirectUpload = ({ onSuccess }: Params = {}) => {
     [handleFileUploadChange, onSuccess]
   );
 
-  const uploading = useMemo(() => uploads.some((upload) => upload.status === 'uploading'), [uploads]);
+  const isUploading = useMemo(() => uploads.some((upload) => upload.status === 'uploading'), [uploads]);
 
   return {
     upload,
     uploads,
-    uploading,
+    isUploading,
+    resetUploads: () => setUploads([]),
+    removeUpload: (id: number) => setUploads(uploads.filter(u => u.id !== id))
```

Comment:
just curious: what's id here?

Reply:
id is this taskId that increments for each upload. https://github.com/DogParkLabs/react-native-activestorage/blob/master/src/lib/directUpload.ts#L15

I can switch to doing this by index instead but I liked that it was some sort of identifier.